dashboard/app/models/census/census_summary.rb

```diff
@@ -146,13 +147,138 @@ def self.decide_teaches(posterior)
   end
 
   def self.summarize_school_data(school, school_years, years_with_ap_data, years_with_ib_data, state_years_with_data)
+    summarize_school_data_simple(school, school_years, years_with_ap_data, years_with_ib_data, state_years_with_data)
```

Comment:
Maybe worth passing a hash with this many parameters?

Reply:
Changed